### PR TITLE
feat/Activate refund + pass filter-hook to order-management-plugin

### PR DESCRIPTION
**Pull request by ssolders**
Refund functionality must be activated by the payment-gateway plugin.
SInce we want to handle refunds in the order-management-plugin we only activate the functionality, implement the process_refund hook and then pass on a add_filter-hook that the order-management-plugin will catch.

### CODE REVIEW

File: changelog.readme [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.2.2
```
Comment (perjo927) [inline changelog.readme:1]: Since it's a feature: then we could bump to 1.3.0

**Review by perjo927 — COMMENTED** [review]
LGTM! 

maybe change the version though

File: changelog.readme (hunk 1 above)
Comment (ssolders) [reply changelog.readme]: Good point, changed to 1.3.0

**Comment by perjo927** [comment]
@ssolders  ready to merge?

**Review by perjo927 — APPROVED** [review]

**Pull request merged**